other/materials_designer/create_defect.ipynb

```diff
@@ -0,0 +1,495 @@
+{
```

Comment:
<p>This notebook should not exist - you can keep it in a separate backup branch, but not in something we will merge</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>